Microsoft.Maker.win10/Microsoft.Maker.Serial/Microsoft.Maker.Serial.vcxproj.filters

```diff
@@ -10,11 +10,13 @@
     <ClCompile Include="pch.cpp" />
     <ClCompile Include="..\..\source\Serial\BluetoothSerial.cpp" />
     <ClCompile Include="..\..\source\Serial\USBSerial.cpp" />
+    <ClCompile Include="..\..\source\Serial\DfRobotBleSerial.cpp" />
```

Comment:
I spoke with the DF Robot crew. The "Serial Port" service used is defined by DF Robot and our file is specific to their implementation, so they are happy with the naming convention.